main.cpp

```diff
@@ -325,6 +348,11 @@ void StuffOnMyDesk::cleanDesk()
     keyboard.caseColor = "shiny black";
 }
 
+void StuffOnMyDesk::keyboardCaseColorCout()
```

Comment:
Name things what they are. 
Name functions based on what they do, and make sure that the name includes a verb
- [ ] revise